src/com/goide/GoModuleBuilder.java

```diff
@@ -58,5 +60,8 @@ public boolean isSuitableSdkType(SdkTypeId sdkType) {
   @Override
   public void moduleCreated(@NotNull Module module) {
     CompilerWorkspaceConfiguration.getInstance(module.getProject()).CLEAR_OUTPUT_DIRECTORY = false;
+    ModifiableRootModel rootModel = ModuleRootManager.getInstance(module).getModifiableModel();
+    rootModel.inheritSdk();
+    ApplicationManager.getApplication().runWriteAction(rootModel::commit);
```

Comment:
Yeah, it seems a little hack here, `com.intellij.ide.util.projectWizard.JavaModuleBuilder#setupRootModel` need `myJdk` to be null in order to inherit SDK, but without a concrete SDK, a module can not be created.